src/components/header/header.ts

```diff
@@ -0,0 +1,82 @@
+import { Routes } from '../../models';
+import { createElem } from '../../utilities';
+import './header.scss';
+
+export default class Header {
+  clientTotalSum: HTMLElement;
+  clientBasketNum: HTMLElement;
+
+  constructor() {
+    this.clientTotalSum = createElem('span', 'total__sum');
+    this.clientBasketNum = createElem('div', 'basket__num');
+  }
+
+  drawLayout(): void {
+    const header: HTMLElement | null = document.querySelector('.header');
+    const headerContainer: HTMLElement = createElem('div', 'container header__container', header);
+    const headerEl: HTMLElement = createElem('div', 'header__block', headerContainer);
+
+    const logoBlock: HTMLElement = createElem('div', 'logo', headerEl);
+    createElem('div', 'logo__image', logoBlock);
+    createElem('h1', 'logo__title', logoBlock, 'Online Store');
+    logoBlock.addEventListener('click', () => this.buildPage(Routes.Main));
```

Comment:
**never** use such events on `div` `h1` and other elements. If you want it to be a link, use `a`